service.go

```diff
@@ -0,0 +1,330 @@
+package main
```

Comment:
What are we going to do with this file?

Reply:
Done updated added main.go. and fixes structured.